Insightfinder dashboard json (#1376)
DataDog/integrations-extras: insightfinder/README.md

[InsightFinder][1] AIOps identifies system and application issues before they impact users. Powered by unsupervised machine learning, InsightFinder continuously learns from events, logs, metrics, and changes to detect anomalies, predict incidents, and remediate outages.

InsightFinder ingests data from Datadog through standard APIs and finds anomalous events before your business is impacted.

> Alerts for these anomalous events can then be sent back to Datadog for notification to your team.
> This bi-directional integration provides a complete solution to you IT operations need with advanced AIOps capabilities.

Reviewer: ```suggestion
Alerts for these anomalous events can be sent to notify your team. This bidirectional integration provides advanced AIOps capabilities.
```
